input.fs

```diff
@@ -218,8 +218,7 @@ let (|Mouse|Special|Normal|ImeEvent|TextInput|Unrecognized|) (x: InputEvent) =
     |  Key(_, Key.Add)                                            -> Special("kPlus")
     |  Key(_, Key.Subtract)                                       -> Special("kMinus")
     |  Key(_, Key.Divide)                                         -> Special("kDivide")
-    // TODO|  Key(_, Key.Decimal) -> ???
```

Comment:
This looks to be covered by line 187.

Reply:
must've been abandoned in the first run, then added later ;)